code/modules/experisci/experiment/experiments.dm

```diff
@@ -321,11 +321,11 @@
 	. = ..()
 	damage_percent = rand(15, 95)
 	//updating the description with the damage_percent var set
-	description = "Your exosuit fabricators allow for rapid production on a small scale, but the structural integrity of created parts is inferior to those made with more traditional means. Damage a few exosuits to exactly [damage_percent]% integrity and scan them to help us determine how the armor fails under stress."
+	description = "Your exosuit fabricators allow for rapid production on a small scale, but the structural integrity of created parts is inferior to those made with more traditional means. Damage a few exosuits to around [damage_percent]% integrity and scan them to help us determine how the armor fails under stress."
 
 /datum/experiment/scanning/random/mecha_damage_scan/final_contributing_index_checks(atom/target, typepath)
 	var/found_percent = round(target.get_integrity() / target.max_integrity, 0.01) * 100
```

Comment:
The math in this line is what caused the issue. Something to do with floating point division. Replacing it with ```var/found_percent = round((target.get_integrity() / target.max_integrity) * 100) //Intentionally flooring the result, rather than rounding up if possible``` solved the issue when I tested.

I don't mind the change to a range too much, but possibly it could be limited to +/-2%. That would give you a range of 5% in total.

Reply:
I've implemented your suggestion. Thanks.